### DISCUSSION TITLE
Please help us welcome volunteers

### DISCUSSION THREAD

**Discussion by Navid200** [NightscoutFoundation/xDrip#3598] [General]
@wirbel-at-vdr-portal 

Thanks for your interest in xDrip.

I have diabetes and use xDrip.  My life depends on it.
I am more than happy to help answering any questions you may have about xDrip.

xDrip is maintained by volunteers.  I am one of them.  We are really shorthanded.
So, when someone opens a submission, I want them to feel welcome.

If someone opens a submission and their submission has a problem because it is going to negatively affect xDrip or its maintenance, I will encourage everyone to bring it up in the submission.

However, questioning a submission because of the acronyms is not the way to welcome someone.
The person who needs to understand those acronyms is not the entire world.  You may disagree and you are free to run your repository the way you wish.  If I come to your repository and tell you what to do, I won't be surprised if you disagree as it is your repository.  

Can you help us with xDrip?
May I ask what your interest in xDrip is?  Have you ever used it?

**Comment by Navid200** [comment]
OK, I see your post about the sensor expiry on a follower.
Thanks for that.  

We will not be able to improve xDrip and solve such problems only on our own.  We need every help we can get.
If someone opens a PR, I want them to feel welcome.  Would you please not ask people to explain their PRs?  They are not submitting them to you.
They are being submitted to the owner of this repository.

**Comment by wirbel-at-vdr-portal** [comment]
>> We will not be able to improve xDrip and solve such problems only on our own. We need every help we can get.

I tried to help, i offered it openly.
The problem reported itself was not on my own. I personly dont need xdrip, only my beloved.

>> If someone opens a PR, I want them to feel welcome. Would you please not ask people to explain their PRs? 

No longer going to. It's a pity, that such a genious projects isnt seek any help from others.

>> xDrip is maintained by volunteers. I am one of them. We are really shorthanded.
>> So, when someone opens a submission, I want them to feel welcome.

Doesn't look as you're short-handed. Doesnt look as you're feeling welcome anyone. if you openly inappropriate such to me as offering a volunteer.

I asked to help, really help. As someone doing open-src projects and wiki admins > 20 yrs


I'm silent now. I don't bother you any longer, no bug reports etc.

**Reply by Navid200** [reply]
I myself am not a fan of acronyms.   

AAPS is a sister project: https://github.com/nightscout/AndroidAPS
WIP is work in progress.  
Basal and temp basal are two of insulin injection types a closed loop system can make.
NS stands for Nightscout.  
<br/>  
<br/>  

I'm sorry you were offended.  
The PRs have a purpose.  We use PRs to talk about the code and the impact of the PR on xDrip.  Everything else should go to discussions.  
The issues have a purpose.  
I ask everyone to respect the procedures we have adopted over the years so that we can keep going smoothly.
If you open a PR and someone posts in it telling you to change the words you have used, I will tell them to take it to discussions as well.

**Comment by wirbel-at-vdr-portal** [comment]
Pls, don't teach me for PRs

Unsubscribing now.
